CONTRIBUTING.md

```diff
@@ -1 +1,9 @@
+# Contributing to StopModReposts
+
+A big welcome and thank you for considering contributing to the StopModReposts open source project! It’s people like you that make it a reality for users in our community.
+
+When contributing you must agree to the following terms:
```

Comment:
```suggestion
When contributing you must agree to the following terms:  
```
Just noticed this one. By the way to do new lines in markdown you need two spaces